apps/web-mzima-client/src/app/post/post-routing.module.ts

```diff
@@ -12,7 +12,7 @@ const routes: Routes = [
   {
     path: 'create/:type',
     component: PostEditComponent,
-    data: { breadcrumb: 'Create Post' },
+    data: { breadcrumb: 'Add New Post' },
```

Comment:
We can just have another ticket to add translation for the breadcrumb text in this file, similar to how it's done in app routing module file @MMwandigha

Reply:
@Ifycode Oh yeah its true... I also thought of the same. I noticed that all the breadcrumb texts in the post-routing module haven't been translated yet. I shall create one

Reply:
👍🏼

Reply:
Here's the associated PR for this:
https://github.com/ushahidi/platform-client-mzima/pull/1483